cupy/cuda/runtime.pyx

```diff
@@ -234,6 +236,21 @@ cpdef int deviceGetAttribute(int attrib, int device) except? -1:
     check_status(status)
     return ret
 
+cpdef int deviceGetByPCIBusId(str pci_bus_id) except? -1:
+    # Encode the python string before passing to native code
+    byte_pci_bus_id = pci_bus_id.encode("UTF-8")
+    cdef const char* c_pci_bus_id = byte_pci_bus_id
+
+    cdef int device
+    status = cudaDeviceGetByPCIBusId(&device, c_pci_bus_id)
+    check_status(status)
+    return device
+
+cpdef unicode deviceGetPCIBusId(int device):
+    cdef char pci_bus_id[13]
```

Comment:
How did you determine the number 13?

Reply:
I got this from the CUDA documentation, I'll put a comment addressing this. 
https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__DEVICE.html#group__CUDART__DEVICE_1gea264dad3d8c4898e0b82213c0253def

Reply:
Thanks!